src/GraphBLAS-sharp.Backend/Algorithms/PageRank.fs

```diff
@@ -0,0 +1,203 @@
+﻿namespace GraphBLAS.FSharp.Backend.Algorithms
+
+open GraphBLAS.FSharp
+open GraphBLAS.FSharp.Backend
+open Brahma.FSharp
+open GraphBLAS.FSharp.Objects
+open GraphBLAS.FSharp.Backend.Quotes
+open GraphBLAS.FSharp.Backend.Vector.Dense
+open GraphBLAS.FSharp.Objects.ClMatrix
+open GraphBLAS.FSharp.Objects.ClContextExtensions
+open GraphBLAS.FSharp.Objects.ArraysExtensions
+open GraphBLAS.FSharp.Objects.ClCellExtensions
+
+module internal PageRank =
+    let alpha = 0.85f
+    let accuracy = 0.00000001f
+
+    let countOutDegree (clContext: ClContext) workGroupSize =
+
+        let one =
+            <@ fun (x: float32 option) (_: int option) ->
+                let mutable res = 0
+
+                match x with
+                | Some _ -> res <- 1
+                | None -> ()
+
+                if res = 0 then None else Some res @>
```

Comment:
`IsNone` is not supported in Brahma

Reply:
@IgorErin  I'm not sure that I see an alternative way rather that to support ```IsNone``` translation in Brahma.

Reply:
@gsvgit I'm thinking about a standard library for quotas in Brahma. Because our pattern is special, mutable. Here you can, for example, `mapAssign`. And you won't have to hardcode into the compiler.